package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@digitalbazaar/credentials-v2-context",
+  "version": "0.0.1-0",
+  "description": "A Verifiable Credentials v2 context library for JavaScript.",
+  "main": "js/index.js",
+  "module": "./dist/context.esm.js",
+  "files": [
+    "contexts",
+    "dist/context.js",
+    "js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/credentials-v2-context"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/credentials-v2-context/issues"
+  },
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
+  "homepage": "https://github.com/digitalbazaar/credentials-v2-context#readme",
+  "license": "BSD-3-Clause",
+  "devDependencies": {
+    "@babel/core": "^7.11.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.11.0",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/runtime": "^7.11.0",
+    "babel-loader": "^8.1.0",
+    "chai": "^4.2.0",
+    "cross-env": "^7.0.2",
+    "eslint": "^7.8.1",
+    "eslint-config-digitalbazaar": "^2.5.0",
+    "json-loader": "^0.5.7",
+    "karma": "^5.1.1",
+    "karma-babel-preprocessor": "^8.0.1",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-mocha": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^4.0.2",
+    "mocha": "^8.1.3",
+    "rollup": "^2.26.10",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "build": "npm run clean && npm run serialize && webpack && rollup -c",
+    "clean": "rm -rf dist/* && rm -rf contexts/*.jsonld",
```

Comment:
This is weird, why are we doing this? Why are we checking in a file and then auto-generating it on build? Is this something that's used when just doing a `git clone` w/o a build?

Reply:
It looks like this removes the dist dir (just to be extra-sure) and then generates a new file in the dist dir.